Week3/script.js

```diff
@@ -0,0 +1,273 @@
+/* *** String! *** */
+
+// 1.1
```

Comment:
Could you split up your homework in separate files?

See this:

> For all the following exercises create a new .js file.

Reply:
oo I'm sorry for that. Should I do new pull request?

Reply:
No, no problem!